api/v1/helpers/verbs/doPost.js

```diff
@@ -44,10 +45,27 @@ function doPost(req, res, next, props) {
     postPromise = u.getUserNameFromToken(req,
       featureToggles.isFeatureEnabled('enforceWritePermission'))
       .then((user) => redisModelSample.postSample(req.swagger.params, user));
+  } else if (featureToggles.isFeatureEnabled('enforceWritePermission') &&
+    props.modelName === 'Sample') {
+    postPromise = u.createSample(req, props);
```

Comment:
when enforceWritePermission is enabled and the modelName is "Sample", the control goes to line 50 and there is nothing there to include the createdBy field. Was this intentional?